Resources/public/js/adapter/fancytree.js

```diff
@@ -13,8 +13,14 @@ var FancytreeAdapter = function (options) {
         throw 'The FancytreeAdapter requires a request option.';
     }
 
+    var alwaysTrueFunction = function (node, data) { return true; };
     this.requestData = options.request;
     this.rootNode = options.root_node || '/';
+    this.dndOptions = jQuery.extend({
+        enabled: false,
+        isNodeDraggable: alwaysTrueFunction,
```

Comment:
this option specifies a function that can determine if a node can be dragged (e.g. based on the FQCN).